integration_tests/test_qbs_plugin.py

```diff
@@ -0,0 +1,38 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Dan Chapman <dpniel@ubuntu.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import os
+
+import testscenarios
+import integration_tests
+
+
+class QbsPluginTestCase(testscenarios.WithScenarios,
+                        integration_tests.TestCase):
+
+    scenarios = [
+        ('qbs-nil', dict(project_dir='qbs-nil')),
+        ('qbs-qt', dict(project_dir='qbs-qt')),
+    ]
+
+    def test_stage_qbs_plugin(self):
+
+        self.run_snapcraft('stage', self.project_dir)
+
+        # The test binaries have the same name as the project dir
+        binary_output = self.get_output_ignoring_non_zero_exit(
+            os.path.join(self.stage_dir, 'bin', self.project_dir))
+        self.assertNotEqual('Wrong snapcraft string', binary_output)
```

Comment:
sorry, I meant to build them again with project.printString, to make sure that those arguments work.